MSALiOSB2C/ViewController.swift

```diff
@@ -353,26 +292,39 @@ class ViewController: UIViewController, UITextFieldDelegate, URLSessionDelegate
     override func viewWillAppear(_ animated: Bool) {
         
         if self.accessToken.isEmpty {
-            
-            signoutButton.isEnabled = false;
-            callGraphApiButton.isEnabled = false;
-            editProfileButton.isEnabled = false;
-            refreshTokenButton.isEnabled = false;
-            
-            
+            signoutButton.isEnabled = false
+            callGraphApiButton.isEnabled = false
+            editProfileButton.isEnabled = false
+            refreshTokenButton.isEnabled = false
         }
     }
     
-    func getUserByPolicy (withUsers: [MSALUser], forPolicy: String) throws -> MSALUser? {
+    func getAccountByPolicy (withAccounts: [MSALAccount], policy: String) throws -> MSALAccount? {
         
-        for user in withUsers {
-            if (user.userIdentifier().components(separatedBy: ".")[0].hasSuffix(forPolicy.lowercased())) {
-                return user
+        for account in withAccounts {
+            if (account.homeAccountId != nil && account.homeAccountId!.objectId!.hasSuffix(policy.lowercased())) {
```

Comment:
nit: this won't work for multi account cases. Can we add a comment saying this sample is a single account sample and for multi-account apps they would also need to check the object Id part.

Reply:
make sense, added some comments about the structure of the b2c object id.